NitroxModel/DataStructures/AbsoluteEntityCell.cs

```diff
@@ -1,69 +0,0 @@
-﻿using System;
-using NitroxModel.Helper;
-
-namespace NitroxModel.DataStructures
-{
-    [Serializable]
-    public class AbsoluteEntityCell
```

Comment:
Did you mean to remove this?

Reply:
Yes, see the first commit. The file has been modified and moved; this original version was reintroduced by a pull request.

Reply:
My bad...